public_html/js/index.js

```diff
@@ -1,104 +1,120 @@
-/**
- * Save a review
- * @param id int ID of the record
- * @param val string Save value 'fixed' or 'false'
- */
-function saveState( id, val ) {
-	var buttonId, unusedButtonId, buttonClass, unusedButtonClass;
+$( document ).ready( function() {
+	/** Listeners */
+	$('body').tooltip({
+		selector: '[data-toggle="tooltip"]'
+	});
+	$( '.records' ).on( 'click', '.js-save-state', function () {
+		saveState( $( this ).data( 'id' ), $( this ).data( 'status' ) );
+	} );
+	$( '.js-load-more' ).on( 'click', loadMoreResults );
+	$( '.js-compare-button' ).on( 'click', function () {
+		// pass the dataset of the element as an object to toggleComparePane
+		toggleComparePane.call(this, this.dataset);
```

Comment:
@Niharika29 I did forget to do `toggleComparePane`, here it is. Since there are so many values we need it seems easier to just pass it in as a key/value hash